pkg/webhook/resources/setting/validator.go

```diff
@@ -404,6 +411,77 @@ func (v *settingValidator) customizeTransport() error {
 	return nil
 }
 
+func validateNTPServers(setting *v1beta1.Setting) error {
```

Comment:
`validateNTPServers` needs to be registered with the settings validation handler at: https://github.com/harvester/harvester/blob/master/pkg/webhook/resources/setting/validator.go#L106-L114

currently this is not being called and I am able to submit incorrect ntp server settings: 

```shell
time="2023-05-28T23:54:52Z" level=info msg="Processing setting ntp-servers, value: 192.168.122.1"
time="2023-05-28T23:54:52Z" level=error msg="error syncing 'ntp-servers': handler harvester-setting-controller: failed to parse NTP settings: invalid character '.' after top-level value, requeuing"
```

Reply:
please ignore, should have updated the webhook image during testing.